soc/arm/nordic_nrf/nrf51/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
+# NB: The define 'NRF51822' is deprecated. 'NRF51' should be used in
+# it's place.
 zephyr_compile_definitions_ifdef(
   CONFIG_SOC_SERIES_NRF51X
-  NRF51
   NRF51822
```

Comment:
Now it looks a bit weird. Why `NRF51822` is to stay here? I think it should be also moved. Or better removed. Is it actually used anywhere?

Reply:
There are no known in-tree users.

I can remove it. I believe it was added by mistake and likely never used out-of-tree.

Reply:
Removed.